test/e2e/resources/components/roAssociativeArray.brs

```diff
@@ -11,6 +11,7 @@ sub main()
     print "can delete elements: " aa.delete("baz")              ' => true
     print "can look up elements: " aa.lookup("foo") = "foo"     ' => true
     print "can look up elements (brackets): " aa["foo"] = "foo" ' => true
+    print "can case insensitive look up elements: " aa.lookupCI("foO") = "foo" ' => true
```

Comment:
Can we add another call to `lookupCI` after we call `setModeCaseSensitive` as well?

Reply:
done